JustFakeIt/MatchingStrategy/BodyMatching/PartialJsonMatching.cs

```diff
@@ -0,0 +1,52 @@
+﻿using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+
+namespace JustFakeIt
+{
+    public class PartialJsonMatching : IBodyMatchingOption
+    {
+        public bool MatchBody(string expectedBody, string actualBody)
+        {
+            return string.IsNullOrEmpty(expectedBody) || actualBody.Equals(expectedBody) || JsonKeyValuesMatch(expectedBody, actualBody);
```

Comment:
This is going to throw a wobbly if the `actualBody` is null. Shouldn't happen, but probably worth making this class rock solid.